### PR TITLE
Add macOS install instructions

### PR DESCRIPTION
**Pull request by yurikoles**
Signed-off-by: Yurii Kolesnykov <root@yurikoles.com>

### CODE REVIEW

**Comment by badabing2005** [comment]
Thanks,

I appreciate that you're making it easier for power users to install it from command line, however:

Don't get me wrong, but why would this be MacOS install instructions?
Any MacOS user should know how to install dmg, no? 
On the other hand many MacOS users who are not power users don't know anything about brew and how to properly set it up.
I see more issues with this approach.
- For starters the command won't work without first knowing to do `brew tap yurikoles/yurikoles`
- Then it is reliant on you to quickly update on every release.
- And most importantly, security, I'm not suggesting that you're not trustworthy, however if you were or if your Github account is compromised, this would be one way to fool users.

I might add a footnote with a warning to inform users that there are package manager install options available for various platforms and that it is up to the user to verify the authenticity and recency of the packages.

**Comment by yurikoles** [comment]
> * For starters the command won't work without first knowing to do `brew tap yurikoles/yurikoles`

That's wrong, the command I provided contains tap name inside, so it'll be auto-tapped on first execution.

**Comment by yurikoles** [comment]
When I first created this cask `brew audit` complained that this GH repo isn't too famous because of lack of likes and forks. So I put it in my personal tap.

Now I see this repo gained a lot of popularity, so it may be qualified for inclusion in the official `homebrew/cask` repo, let me try again.

They have an `autobump`, so updates shouldn't lag for more than a couple of hours.

**Comment by yurikoles** [comment]
I'm sorry, the issue is different, it's notarization:

```
audit for pixelflasher: failed
 - Signature verification failed:
/private/tmp/cask-audit20240806-96722-bgvhna/PixelFlasher.app: rejected

macOS on ARM requires software to be signed.
Please contact the upstream developer to let them know they should sign and notarize their software.
/opt/homebrew/Library/Taps/homebrew/homebrew-cask/Casks/p/pixelflasher.rb: * line 5, col 2: Signature verification failed:
    /private/tmp/cask-audit20240806-96722-bgvhna/PixelFlasher.app: rejected
    
    macOS on ARM requires software to be signed.
    Please contact the upstream developer to let them know they should sign and notarize their software.
Error: 1 problem in 1 cask detected.
```

I can help you to set up a notarization in GHA.

**Comment by badabing2005** [comment]
Thanks,
I don't have a key, are they available for free?

**Comment by yurikoles** [comment]
A $99/y membership is [required](https://forums.developer.apple.com/forums/thread/121113) for that.

**Comment by yurikoles** [comment]
Please consider creating your own tap that will be controlled by you, but others can contribute to. Could you please create some repository, e.g `badabing2005/homebrew-PixelFlasher`*? And I'll create a PR with cask code from my tap.

*`homebrew-` at the start of the repo is needed as homebrew naming convention, please don't remove it.

**Comment by yurikoles** [comment]
So let's do your tap, so the security and update speed question will be solved.

**Comment by badabing2005** [comment]
https://github.com/badabing2005/homebrew-PixelFlasher

**Comment by yurikoles** [comment]
I sent a [PR](../../homebrew-PixelFlasher/pull/1).

**Comment by badabing2005** [comment]
Done

**Comment by yurikoles** [comment]
I forgot that `*.rb` in tap root is treated as formulas, so I moved the cask to the `Casks` subdirectory.

**Comment by yurikoles** [comment]
This PR was rebased with a new tap name.

**Comment by badabing2005** [comment]
> I forgot that `*.rb` files in tap root are treated as formulas, so I moved the cask to the `Casks` subdirectory.

I'm not sure I follow,
where did you move the .rb into Casks directory?
You submitted a PR or should I be creating the directory and moving it there?

**Review by badabing2005 — APPROVED** [review]

**Comment by yurikoles** [comment]
You forgot to trim 'v' from the tag again.

https://github.com/badabing2005/homebrew-PixelFlasher/blob/7e9d32cc09901923780025465bfda9af8e23382e/Casks/pixelflasher.rb#L2


And commit message actually says that: [Update PixelFlasher to version vv7.4.2.2](https://github.com/badabing2005/homebrew-PixelFlasher/commit/7e9d32cc09901923780025465bfda9af8e23382e).

**Comment by yurikoles** [comment]
I suggest you add some checks to your workflow:
```sh
brew style --cask badabing2005/pixelflasher/pixelflasher
brew audit --cask --online badabing2005/pixelflasher/pixelflasher
```

**Comment by yurikoles** [comment]
Although `brew style` works with local paths, `brew audit` works only with names, so for it to work you'll need to mimic `brew tap` by something like:

```sh
mkdir -p $(brew --repository)/Library/Taps/badabing2005
ln -s <path-to-homebrew-PixelFlasher> $(brew --repository)/Library/Taps/badabing2005/
```

**Comment by badabing2005** [comment]
Fixed the root issue, check is not added.
I don't think I'd be touching this much again, now that it is working fine.
